packages/vue-hooks/types/shared.type.ts

```
@@ -0,0 +1,15 @@
+export interface ISharedRenderlessFunctionParams<T> {
+  vm: any
+  props: any
+  slots: any
+  emit: Function
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)